src/CoreBundle/Controller/Backend/SupportMetaModelsController.php

```diff
@@ -60,14 +62,14 @@ class SupportMetaModelsController
     /**
      * Create a new instance.
      *
-     * @param EngineInterface     $templating The twig engine.
+     * @param Environment         $twig       The twig engine.
```

Comment:
Either also use `TwigEnvironment` here or replace the alias in the imports above (See [build error](https://travis-ci.org/MetaModels/core/jobs/601263989#L410))